cmd/xl-v1-multipart.go

```diff
@@ -361,7 +361,7 @@ func (xl xlObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 	tmpPartPath := path.Join(tmpPart, partSuffix)
 
 	// Delete the temporary object part. If PutObjectPart succeeds there would be nothing to delete.
-	defer xl.deleteObject(ctx, minioMetaTmpBucket, tmpPart)
+	defer xl.deleteObject(ctx, minioMetaTmpBucket, tmpPart, len(xl.getDisks())/2+1, false)
```

Comment:
same as above. We should use `writeQuorum` in place of `len(xl.getDisks()) / 2 + 1`.